packages/theme-predictive-search/src/utilities/Cache.js

```diff
@@ -0,0 +1,39 @@
+export default function Cache(config) {
+  this._store = {};
```

Comment:
What is the expected size of this store? Is it a threat to memory?

Reply:
I did want to limit the memory used by this cached results. Do you have some thoughts around that pattern?